frontend/public/components/modals/configure-count-modal.tsx

```diff
@@ -32,7 +32,9 @@ export const ConfigureCountModal = withHandlePromise((props: ConfigureCountModal
     e.preventDefault();
 
     const patch = [{ op: 'replace', path, value: _.toInteger(value) }];
-    const opts = { path: 'scale' };
+    // Jobs are not using the `/scale` endpoint.
```

Comment:
It would be better to pass this as a prop to the dialog if possible instead of hard-coding kinds here.